structuretoolkit/visualize.py

```diff
@@ -807,3 +809,78 @@ def _get_flattened_orientation(
     flattened_orientation[:3, :3] = _get_orientation(view_plane)
 
     return (distance_from_camera * flattened_orientation).ravel().tolist()
+
+
+def plot_isosurface(
+    mesh,
+    value,
+    cell: Optional[Union[Atoms, list, np.ndarray, float]] = None,
+    structure_plot: Optional["plotly.graph_objs._figure.Figure"] = None,
+    isomin: Optional[float] = None,
+    isomax: Optional[float] = None,
+    surface_fill: Optional[float] = None,
+    opacity: Optional[float] = None,
+    surface_count: Optional[int] = 5,
+    colorbar_nticks: Optional[int] = None,
+    caps: Optional[dict] = dict(x_show=False, y_show=False, z_show=False),
+    colorscale: Optional[str] = None,
+    height: Optional[float] = 600,
```

Comment:
```suggestion
    height: float = 600.,
```

Unless this is height in (integer) pixels and the type hint was what was wrong